### DISCUSSION TITLE
The current behavior after modifying a cell and clicking outside of it is not the same as in Notion, should we implement Notion's behavior instead?

### DISCUSSION THREAD

**Discussion by bosiraphael** [twentyhq/twenty#1833] [Engineering vision]
Right now, when modifying a cell in Twenty and clicking outside of it, the cell goes back to its previous value.
In Notion, clicking outside of the cell saves the new value (same behavior as clicking on Enter).

If we enter a value with the wrong type in a cell in Twenty, the cell goes back to its previous value, whereas in Notion, the cell becomes empty.

Should we change these two behaviors ?

Twenty :

https://github.com/twentyhq/twenty/assets/71827178/4b7e2743-3137-48ca-95ce-f57fb27655e8

Notion :

https://github.com/twentyhq/twenty/assets/71827178/0d5ba6c0-1f8f-449b-89c0-a2e1121b3696

**Comment by lucasbordeau** [comment]
@Bonapara What are your thoughts ?

**Reply by Bonapara** [reply]
I propose implementing the first behavior (Well noted, @bosiraphael!).I recommend maintaining the second behavior as it is.
